java/ingest/ingest-core/src/main/java/sleeper/ingest/task/IngestTaskStatusesBuilder.java

```diff
@@ -24,9 +24,20 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+/**
+ * A helper class for constructing multiple ingest task statuses.
```

Comment:
We can omit "class" here. See the Oracle style guide, "Class/interface/field descriptions can omit the subject and simply state the object": https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html